crates/sncast/src/helpers/token.rs

```diff
@@ -0,0 +1,29 @@
+use clap::ValueEnum;
+use serde::Serialize;
+use starknet::macros::felt;
+use starknet_types_core::felt::Felt;
+
+const STRK_CONTRACT_ADDRESS: Felt =
+    felt!("0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d");
+
+const ETH_CONTRACT_ADDRESS: Felt =
+    felt!("0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7");
+
+#[derive(Default, Serialize, Clone, Copy, Debug, ValueEnum, strum_macros::Display)]
+pub enum Token {
+    #[default]
+    #[strum(serialize = "strk")]
```

Comment:
Can't we use do this with plain serde? Maybe this is relevant here https://serde.rs/container-attrs.html#rename_all

Reply:
`#[serde(rename_all = "lowercase")]` will make us accept lowercase values, but when converted to string, they will remain as they are. `#[strum(serialize_all = "lowercase")]` achieves what you want.

Buut, we also need `#[serde(rename_all = "lowercase")]` because with `--json` flag we want to keep the token lowercase.